backend/Dockerfile

```diff
@@ -1,8 +1,11 @@
 # syntax=docker/dockerfile:1
 FROM python:3
+
 ENV PYTHONDONTWRITEBYTECODE=1
 ENV PYTHONUNBUFFERED=1
+
 WORKDIR /code
-COPY requirements.txt /code/
-RUN pip install -r requirements.txt
-COPY . /code/
```

Comment:
I would leave this line, this is not as simple as it might seam.
Docker will still copy the contents to the volume, regardless of this being here or not, this is just to make the Docker file more verbose and concise it also is a good practice.

Reply:
I have not found a use for the `COPY . /code/` instruction, and therefore I would like to have it removed. To make it more verbose I can add a comment to the top of the dockerfile describing that this container is supposed to be used with a volume mounted to `/code`. And I could also describe that this is a development container, not intended for use in production. Would this solve the verbosity problem?